.storybook/stories/datagrid/datagrid-column.stories.ts

```diff
@@ -106,6 +111,7 @@ const ColumnFilterTemplate: StoryFn = args => ({
       </clr-dg-column>
       <clr-dg-column [style.width.px]="250">
         <ng-container ${args.hidableColumns ? '*clrDgHideableColumn' : ''}>Number</ng-container>
+        <clr-dg-numeric-filter [clrDgNumericFilter]="numberFilter"></clr-dg-numeric-filter>
```

Comment:
Why not just set `clrDgColType="number"` on the column? I don't think we need a custom filter for the story.

Reply:
@kevinbuhmann you are completly right, that was my thought proccess as well, but wasn't immediately obvious to me from the clarity design website, that `clrDgColType` accepts `number`, it's only mentioned in the API page.
The [built in filters](https://clarity.design/documentation/datagrid/code/built-in-filters) page there is showing the use of `<clr-dg-numeric-filter [clrDgNumericFilter]="numberFilter"></clr-dg-numeric-filter>`, but not `clrDgColType="number"` which is kind of confusing.

To me the use of `clr-dg-numeric-filter` points to a custom filter situation, than built in.

Reply:
Thanks, updated in https://github.com/vmware-clarity/ng-clarity/pull/1458/commits/15eb5ea8e316bed2a1e82269fcca9e5f6aea1988